### DISCUSSION TITLE
Search is unavailable

### DISCUSSION THREAD

**Discussion by Shizanit** [misskey-dev/misskey#10705] [Q&A] [answered]
For some reason the search stopped working and this error appeared, can you tell me what's wrong?
![image](https://user-images.githubusercontent.com/65107731/233990050-6f1c45ed-6e79-418c-9c4d-e92eee29c73a.png)
![image](https://user-images.githubusercontent.com/65107731/233990119-e00284d6-fdb5-4ee7-b662-0393f1ada4f4.png)
![image](https://user-images.githubusercontent.com/65107731/233990155-89d69854-0a84-45cd-850f-5cd13f9a8ae8.png)

**Comment by mappi-pr** [comment, marked as answer]
@Kitanit 
{your domain}/admin/roles
Is "Usage of note search" Enable ?
![image](https://github.com/misskey-dev/misskey/assets/75538777/27c5b13e-5365-4058-a4c2-743ceb274310)
